packages/admin-ui/src/Link/Link.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { makeDecoratable } from "@webiny/react-composition";
+import { cva, type VariantProps } from "~/utils";
+import {
+    Link as WebinyReactRouterLink,
+    type LinkProps as WebinyReactRouterLinkProps
+} from "@webiny/react-router";
+
+const linkVariants = cva("font-sans rounded-xs", {
+    variants: {
+        size: {
```

Comment:
I'm not sure we need a `size` variant: the font size of a link should be inherited from its parent; otherwise, we would end up in this situation: 

`<Text size="lg">Lorem ipsum with <Link size="lg">link</Link> text.</Text>`

declaring the `size` prop twice.